conduit/utils/cwlparser.py

```diff
@@ -232,7 +232,8 @@ def create_subdag(self, iteration, i, param_db_update_dict, job_params, job, wf_
                     dag=subdag,
                     queue=step_job_queue,
                     workflow_id=parent_dag_id,
-                    score_format=score_format
+                    score_format=score_format,
+                    pool='Batch'
```

Comment:
if cwl doc is local, why is the pool batch?

Reply:
This might be github showing the changes weird. The `pool=Batch` line is in the AWSBatchOperator section. (Line 236)